openverse_catalog/dags/providers/provider_api_scripts/phylopic.py

```diff
@@ -255,14 +270,28 @@ def _get_image_info(result, _uuid):
         return img_url, width, height, thumbnail
 
 
+def _compute_date_range(date_start: str, days: int = DEFAULT_PROCESS_DAYS) -> str:
+    """
+    Given an ISO formatted date string and a number of days, compute the
+    ISO string that represents the start date plus the days provided.
+    """
+    date_end = date.fromisoformat(date_start) + timedelta(days=days)
+    return date_end.isoformat()
+
+
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="PhyloPic API Job", add_help=True)
     parser.add_argument(
-        "--date",
+        "--date-start",
         default="all",
-        help="Identify all images" " from a particular date (YYYY-MM-DD).",
+        help="Identify all images from a particular date (YYYY-MM-DD).",
```

Comment:
Tiny nit, `from a particular date` reads "images _on that particular date_" to me. Maybe `starting from a particular date`?

Reply:
Oh, I should definitely make this help string better! Thanks for the nudge, I was initially just correcting the spacing on it.

Reply:
![image](https://user-images.githubusercontent.com/10214785/159364037-a56d810e-d69f-49f0-8351-ea0ad5dc5f77.png)
That's a bit clearer! :smile: